parl/remote/client.py

```diff
@@ -96,6 +99,15 @@ def read_local_files(self, distributed_files=[]):
         pyfiles['other_files'] = {}
 
         code_files = filter(lambda x: x.endswith('.py'), os.listdir('./'))
+        
+        parsed_distributed_files = []
+        for distributed_file in distributed_files:
+            parsed_list = glob.glob(distributed_file)
```

Comment:
if the return list of glob.glob is empty, we should raise an exception.

Reply:
fixed